src/evergreen/api.py

```diff
@@ -348,6 +348,16 @@ def project_by_id(self, project_id: str) -> Project:
         url = self._create_url(f"/projects/{project_id}")
         return Project(self._paginate(url), self)  # type: ignore[arg-type]
 
+    def get_human_readable_project_id(self, project_id: str) -> str:
```

Comment:
I don't think this should exist on the api class. We have just been using this class for methods that wrap the actual Evergreen API and with just those, this class is already huge. We should just have this on the `Task` and `Build` objects themselves.